code/modules/spells/aoe_turf/conjure/druidic_spells.dm

```diff
@@ -86,24 +90,36 @@
 			newVars = list("maxHealth" = 200,
 						"health" = 200,
 						"melee_damage_lower" = 30,
-						"melee_damage_upper" = 30,
-						"color" = "#d9d9d9" //basically we want them to look different enough that people can recognize it.
+						"melee_damage_upper" = 30
 						)
 			return "Your bear has been upgraded from a whelp to an adult."
 		if(3)
 			newVars = list("maxHealth" = 250,
 						"health" = 250,
 						"melee_damage_lower" = 45,
-						"melee_damage_upper" = 45,
-						"color" = "#8c8c8c"
+						"melee_damage_upper" = 45
 						)
 			return "Your bear has been upgraded from an adult to an alpha."
 		if(4)
 			newVars = list("maxHealth" = 300,
 						"health" = 300,
 						"melee_damage_lower" = 55,
 						"melee_damage_upper" = 55,
-						"resistance" = 10,
-						"color" = "#0099ff"
+						"resistance" = 10
 						)
 			return "Your bear is now worshiped as a god amongst bears."
+
+/datum/spell/aoe_turf/conjure/summon/bear/proc/spell_learned(mob/M)
+	if(!M || !M.mind)
+		return
+
+	to_chat(M, "A short note regarding the available commands for your bear has been placed in your notes.")
+	var/bear_guide = "<br /><b>Congratulations, you now can summon a magic, bluespace bear.</b><br /> The bear's mind is dominated, so it should not attack you under any circumstances. You can command your bear through words, but remember that it must hear you.<br /><b> Your bear knows the following commands:</b><br />- stay<br /> - stop: your bear will stop following all previous commands.<br /> - attack: designate someone (or everyone!) as a threat. <br /> - follow: your bear will follow anyone designated, for example say 'Bear follow me'.<br />-  dance: it wil literally dance.<br /> - befriend: your bear won't attack those who are designated as friends. An example is 'Bear befriend pig'. If your bear can see the pig, it will remember it. <br />- forget: use to cancel previous command.<br /> Simply pronounce your bear name when issuing a command. Remember to address your bear by name or else it will not heed the command.<br />"
```

Comment:
```suggestion
	var/bear_guide = "<br /><b>Congratulations, you now can summon a magic, bluespace bear.</b><br /> The bear's mind is dominated, so it should not attack you under any circumstances. You can command your bear through words, but remember that it must hear you.<br /><b> Your bear knows the following commands:</b><br />- stay<br /> - stop: your bear will stop following all previous commands.<br /> - attack: designate someone (or everyone!) as a threat. <br /> - follow: your bear will follow anyone designated, for example say 'Bear follow me'.<br />-  dance: it will literally dance.<br /> - befriend: your bear won't attack those who are designated as friends. An example is 'Bear befriend pig'. If your bear can see the pig, it will remember it. <br />- forget: use to cancel previous command.<br /> Simply pronounce your bear name when issuing a command. Remember to address your bear by name or else it will not heed the command.<br />"
```